spec/requests/users_request_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe "Users", type: :request do
+
+  let(:user) { create(:user) }
+  let(:tweet) { create(:tweet, user: user) }
+  let(:reply) { create(:reply, user: user, tweet: tweet) }
+
+  describe "GET /users/:id/tweets" do
+    context "with valid user id" do
+      it "returns a list of user's tweets" do
+        get user_tweets_path(user.id)
```

Comment:
`user_tweets_path` doesn't exists because you did not added an alias to the route in your` routes.rb `file